sdk/core/test/integration/otp/core.test.ts

```diff
@@ -352,6 +358,112 @@ parallel('CommonNetworkMember [OTP]', () => {
       expect(error.name).to.eql('COR-17')
     })
 
+    it('user can make typo in OPT ', async function () {
+      const { inbox } = await createUser()
+      const loginToken = await AffinidiWallet.initiateLogInPasswordless(options, inbox.email, messageParameters)
+      const loginCode = await waitForOtpCode(inbox)
+
+      try {
+        await AffinidiWallet.completeLogInPasswordless(options, loginToken, randomOTP())
+      } catch (errFirstTry) {
+        expect(errFirstTry).to.be.instanceOf(SdkError)
+        expect(errFirstTry.name).to.eql('COR-5')
+      }
+
+      try {
+        await AffinidiWallet.completeLogInPasswordless(options, loginToken, randomOTP())
+      } catch (errFirstTry) {
+        expect(errFirstTry).to.be.instanceOf(SdkError)
+        expect(errFirstTry.name).to.eql('COR-5')
+      }
+
+      await wait(180)
+
+      const commonNetworkMember = await AffinidiWallet.completeLogInPasswordless(options, loginToken, loginCode)
+      checkIsWallet(commonNetworkMember)
+      expect(commonNetworkMember.did).to.exist
+    })
+
+    it('user get COR-13 on 3 wrong otp codes ', async function () {
```

Comment:
But we already have `it('throws COR-13 at the third call to #confirmSignIn with wrong confirmation code'` test here

Reply:
yep but here we make sure that user still able to login . so before test was there but we have seriose bug there . so i reinsure that user are able to do a passwordless login after getting errors and etc